src/main/java/mike/cli/CommandLine.java

```diff
@@ -270,13 +321,13 @@ public static void commandInterface() {
 							System.out.println();							
 						}
 					} else {
-						System.out.println(errorMessage + commandUsage[13] + commandUsage[14] + commandUsage[15] + "\n");
+						System.out.println(errorMessage + commandUsage[16] + commandUsage[17] + commandUsage[18] + "\n");
 					}
 					break;
 				// Calls clear 
 				case "clear":
 					if (commands.length != 1) {
-						System.out.println(errorMessage + commandUsage[16] + "\n");
+						System.out.println(errorMessage + commandUsage[19] + "\n");
 					} else if (!userClasses.empty()) {
 						System.out.println("\nAre you sure you want to delete everything?");
 						System.out.println("Type 'yes' to delete, or 'no' to go back.");
```

Comment:
I actually want to comment on the lines below but it won't let me, so I copy and pasted them here. 
boolean answer = savePrompt(true, cmdLine);
							
						if (!answer) {
							userClasses.clear();
							prompt = true;	
						}
					}
					break;

Reply:
Shouldn't prompt be set to false here? We have already prompted the user to clear and they have said yes. So, they know they have cleared their unsaved changes and now there are no changes left. When you hit quit after calling clear it prompts "you have unsaved changes" when both classes and relationships are empty.

Reply:
That's fair, but those changes are also still "unsaved". I wanted to keep it consistent between things that are 100% saved and written to the file, and those that aren't. I get what you mean though - they confirmed they want to clear, so obviously they want to save that. On the other hand, it isn't _actually_ saved yet, so I wanted them to be aware of that.

If you or anyone feels really strongly that I should change the prompt flag to false here though, we can.